component/loki/process/process.go

```diff
@@ -124,7 +105,6 @@ func (c *Component) Update(args component.Arguments) error {
 		if err != nil {
 			return err
 		}
-		c.processOut = make(chan loki.Entry)
```

Comment:
Same here, we might close c.processOut when calling Update. Don't we want to re-create it here?

Reply:
Moved the close to defer on the run method. Which should work since its only created once.

Reply:
I don't think processOut should never change during the lifecycle of the process component. Outs the output channel for the ll the ApiEventHandler channels/processes.